packages/scikit-image/meta.yaml

```diff
@@ -1,22 +1,22 @@
 package:
   name: scikit-image
-  version: 0.15.0
+  version: 0.18.1
+requirements:
+  run:
+  - numpy
+  - scipy
+  - matplotlib
+  - networkx
+  - pillow
+  - imageio
+  - pywavelets
 source:
   patches:
   - patches/no-openmp.patch
   - patches/unwrap.patch
   - patches/preferred_plugins.patch
-  sha256: df111e654b47e5ea456c50553debe4c5ddd97258894c7ad3b7f2f9f10798e053
-  url: https://files.pythonhosted.org/packages/e0/46/ca035f5d7d3414124a3a5ef22cd2e75c0c5149042a668375f1d44eb69f8f/scikit-image-0.15.0.tar.gz
-requirements:
-  run:
-    - numpy
-    - scipy
-    - matplotlib
-    - networkx
-    - pillow
-    - imageio
-    - pywavelets
+  sha256: fbb618ca911867bce45574c1639618cdfb5d94e207432b19bc19563d80d2f171
+  url: https://files.pythonhosted.org/packages/6e/be/a8ccf9d949a55023cf02438310e068c263ce3dd6bbf31f9229d3db6e551a/scikit-image-0.18.1.tar.gz
```

Comment:
Similarly let's address this in a separate PR. And scikit-image currently cannot be updated as it requires a newer version of scipy.